src/style.css

```diff
@@ -33,3 +42,15 @@ input {
 li h4 {
   padding: 5px;
 }
+
+.width {
+  width: 50%;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
```

Comment:
- [ ] This styling hides the top contents when the page content is populated beyond the view height. Kindly remove the `justify-content` property or change its value. 

![Screenshot from 2021-10-08 15-32-41](https://user-images.githubusercontent.com/64894927/136558669-21e2549d-303f-4874-ac12-14ea421e4dfb.png)